protocol/git/README.md

```diff
@@ -106,6 +106,7 @@ View a list of new commits. View changed files. Merge branch into master.
     git log origin/master..<branch-name>
     git diff --stat origin/master
     git checkout master
+    git pull
```

Comment:
Yeah absolutely, we'd not be able to push if we weren't up to date. I was just thinking about making it less likely for that to happen. `pull --rebase` would work, I tend to use that by default anyway. But I guess if pulling isn't getting us anything either way maybe it's better to leave it out.